jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/compositeid/TestCompositeId.java

```diff
@@ -0,0 +1,143 @@
+/********************************************************************************
+ * Copyright (c) 1998, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1998, 2018 IBM Corporation. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ * Contributors:
+ *     07/09/2018-2.6 Jody Grassel
+ *       - 536853: MapsID processing sets up to fail validation
+ ******************************************************************************/
```

Comment:
glassfish-copyright-plugin will fail to verify this. To void another "batch change" once we add it to the build, I'd suggest to follow the pattern used in other files (or shown in https://github.com/eclipse-ee4j/eclipselink/blob/master/project-admin/Sample.java)

Reply:
@lukasj Fixed

Reply:
Fixed in 2.7 pull request, updating the one for master now.